src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java

```diff
@@ -303,7 +307,7 @@ public UserServiceClient userServiceClient(
     }
 
     @Bean
-    public OtpServiceClient otpServiceClient(@Qualifier("customBuilder") WebClient.Builder builder,
+    public OtpServiceClient otpServiceClient(@Qualifier("customBuilder") Builder builder,
```

Comment:
It will not be obvious, what builder it's. Keep that change.

Reply:
Done.